eng/SharedFramework.Local.props

```diff
@@ -46,6 +46,8 @@
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Authentication.Core" />
     <AspNetCoreAppReference Include="Microsoft.Net.Http.Headers" />
     <AspNetCoreAppReference Include="Microsoft.AspNetCore.Http.Abstractions" />
+    <AspNetCoreAppReference Include="Microsoft.AspNetCore.Http.Api.Abstractions" />
```

Comment:
We don't want new assemblies. Put these in the Http assembly.

Reply:
What assembly do you think MapAction itself belongs in?

Reply:
Microsoft.AspNetCore.Routing

Reply:
👍🏽 This can very well live with other Map* extensions.

Reply:
> Microsoft.AspNetCore.Routing

Ugh, I don't think it belongs in routing. Can we discuss this before we put it there?

Reply:
Where do you think it belongs? (btw we're not making new assemblies)

Reply:
Where do you think it belongs? I've already moved it, but I have no problem moving it again.

Reply:
Lets leave it there and discuss it in the next meeting. I don't want to hold up this PR for that. Especially if it results in making a new assembly.